.env.development

```diff
@@ -8,6 +8,7 @@ ENABLE_REGEX_MATCHING_SYNC_APIS='blood_pressures|facilities|patients|prescriptio
 ENABLE_FIXED_OTP_ON_REQUEST_FOR_QA=false
 ENABLE_AUTO_APPROVE_USER_FOR_QA=false
 ENABLE_USER_ANALYTICS=true
+ENABLE_COUNSELLOR_FOLLOWUPS=true
```

Comment:
This should be `PATIENT_FOLLOWUPS`; we don't want to tie features to a role

Reply:
makes sense